app/components/component-docs/dataDisplay/CoreAvatar.docs.js

```diff
@@ -248,18 +250,20 @@ export default function CoreAvatarDocs() {
               }
             </CoreTypographyBody1>
 
-            <CoreTypographyBody1>
-              - the provided children - the first letter of the alt text - a generic avatar icon
-            </CoreTypographyBody1>
+            { /**
+            * @todo CoreList need to be fix 
+            * the provided children
+              the first letter of the alt text
+              a generic avatar icon
+             */}
           </>
         }
-        code={`
-<CoreStack styleClasses={[CoreClasses.ALIGNMENT.JUSTIFY_CONTENT_CENTER]} direction="row" spacing={2}>
+        code={`<CoreStack styleClasses={[CoreClasses.ALIGNMENT.JUSTIFY_CONTENT_CENTER]} direction="row" spacing={2}>
   <CoreAvatar
     styleClasses={[CoreClasses.BG.BG_PRIMARY]}
     alt="Remy Sharp"
     src="/broken-image.jpg"
-  >B</CoreAvatar>
+  >W</CoreAvatar>
```

Comment:
The requirement is to show the first letter of the alt text will show if the src is broken.